validator/src/main.rs

```diff
@@ -2637,6 +2645,7 @@ pub fn main() {
         accounts_shrink_ratio,
         runtime_config: RuntimeConfig {
             bpf_jit: !matches.is_present("no_bpf_jit"),
+            log_messages_bytes_limit: value_t!(matches, "log_messages_bytes_limit", usize).ok(),
```

Comment:
I think `value_of(&matches, "log_messages_bytes_limit")` should work, and would be a simpler way to write this.

Reply:
Opened PR #26644 to cleanup these issues.